website/source/docs/platform/k8s/run.html.md

```diff
@@ -256,6 +256,32 @@ the clients, set the `client.image` value to the desired Consul version.
 Then, run `helm upgrade`. This will upgrade the clients in batches, waiting
 until the clients come up healthy before continuing.
 
+### Using Existing PVCs
+
+The only way to use a pre-created PVC is to name them in the format Kubernetes expects:
+
+```
+data-<kubernetes namespace>-<release name>-consul-server-<ordinal>
```

Comment:
```suggestion
data-<kubernetes namespace>-<helm release name>-consul-server-<ordinal>
```

Someone I was helping didn't know what release name was so make it clear it's the helm one.